src/renderer/shared/actions/toastsActions.js

```diff
@@ -0,0 +1,32 @@
+export const ENQUEUE = 'ENQUEUE_TOAST';
+export const DEQUEUE = 'DEQUEUE_TOAST';
+
+export const TYPE_SUCCESS = 'success';
+export const TYPE_ERROR = 'error';
+export const TYPE_INFO = 'info';
+
+export const showToast = ({ message, type, autoDismiss = 5 } = {}) => ({
+  type: ENQUEUE,
+  payload: { message, type, autoDismiss }
+});
+
+export const showSuccessToast = ({ message } = {}) => showToast({
+  message,
+  type: TYPE_SUCCESS
+});
+
+export const showErrorToast = ({ message } = {}) => showToast({
+  message,
+  type: TYPE_ERROR,
+  autoDismiss: false
```

Comment:
shouldn't this be configurable `true/false` through passing it in at line 18 as a param?

Reply:
I wavered on this since the raw `showToast` accepts it as a param, but I'll add it for the alternative versions too. 👍